WooCommerce/Classes/POS/Presentation/PointOfSaleDashboardView.swift

```diff
@@ -11,6 +11,10 @@ struct PointOfSaleDashboardView: View {
         self.totalsViewModel = viewModel.totalsViewModel
     }
 
+    private var isCartShown: Bool {
+        !viewModel.itemListViewModel.isEmptyOrError
```

Comment:
nitpick

To answer to this comment: https://github.com/woocommerce/woocommerce-ios/pull/13361#discussion_r1682143021

My suggestion was to move such checks to VM ( `PointOfSaleDashboardViewModel`), so it would be `viewModel.isCartShown`

Reply:
Ah, that makes sense! I've added it as a follow-up on https://github.com/woocommerce/woocommerce-ios/issues/13380, since I'd also like to extract the existing `MockPOSItemProvider` so we can unit test these 🙇